src/app/helptext/system/alert-service.ts

```diff
@@ -113,10 +115,12 @@ export default {
  this service.'),
 
     SNMPTrap_host_placeholder: T('Hostname'),
-    SNMPTrap_host_tooltip: T(''),
+    SNMPTrap_host_tooltip: T('Hostname or IP address of the system that \
+ will receive SNMP trap notifications.'),
 
     SNMPTrap_port_placeholder: T('Port'),
-    SNMPTrap_port_tooltip: T(''),
+    SNMPTrap_port_tooltip: T('UDP port number on the system receiving \
+ SNMP trap notifications. Example: <i>162</i>'),
```

Comment:
If 162 is the default, that should be mentioned here, rather than using it as an example.
Also missing an ending period.
s/Example: 162/The default is 162./